api/controllers/v1/massif/create.js

```diff
@@ -24,20 +24,39 @@ module.exports = async (req, res) => {
   }
 
   // Check params
-  if (req.param('name') === null) {
-    return res.badRequest('You must provide a name.');
-  }
-  if (req.param('descriptionAndNameLanguage') === null) {
-    return res.badRequest('You must provide a description and name language.');
-  }
+
+  const requiredParams = [
+    'name',
+    'description',
+    'descriptionAndNameLanguage',
+    'descriptionTitle',
+    'geogPolygon',
+  ];
+
+  // eslint-disable-next-line consistent-return
+  requiredParams.forEach((requiredParam) => {
+    if (!req.param(requiredParam)) {
+      return res.badRequest(`${requiredParam} parameter must be provided`);
+    }
+  });
```

Comment:
Tu dois pouvoir éviter le `eslint-disable` en itérant d'une autre manière sur les valeurs. Je pense notamment à une boucle while au sein de laquelle tu remplirais un tableau "missingParameters". Après tu fais un `if(missingParameters.length >0)` et là tu renvoie une badRequest avec tous les paramètres manquants 🙂